Makefile

```diff
@@ -4,7 +4,7 @@
 SHELL := bash
 
 GO-VERSION ?= 1.19.3
-CLOJURE-VERSION ?= 1.12.1
+CLOJURE-VERSION ?= 1.12.3
```

Comment:
does the fact that there are no clj source code updates in the PR imply that 1.12.1 -> 1.12.3 had no glojure-relevant clj stdlib changes?

Reply:
That was my guess. 😊